### ISSUE TITLE
ID comparison deprecation warning

### ISSUE THREAD

**Issue by hackmastera** [pulibrary/figgy#3757]
I get a bunch of these when I run `spec/services/orangelight_reindexer_spec.rb` but it doesn't say where in the code I should be looking to make a change. Or do we want to set this global flag?

```
[DEPRECATION] Valkyrie::IDs will always be equal to their string counterparts in 3.0.0. To silence this message, please either compare IDs or set Valkyrie.config.id_string_equality = true.
```

**Comment by tpendragon** [comment]
Yeah, https://github.com/samvera/valkyrie/pull/821 will fix it when the coverage is fixed.

**Comment by hackmastera** [comment]
closed by #3760
